src/microbe_stage/MicrobeAI.cs

```diff
@@ -87,7 +90,8 @@ private void ChooseActions(Random random, MicrobeAICommonData data)
         }
 
         // If nothing is engulfing me right now, see if there's something that might want to hunt me
-        Vector3? predator = GetNearestPredatorItem(data.AllMicrobes)?.Translation;
+        // TODO: https://github.com/Revolutionary-Games/Thrive/issues/2323
+        Vector3? predator = GetNearestPredatorItem(data.AllMicrobes)?.GlobalTransform.origin;
```

Comment:
Are you sure the distance to another colony should be to the center, not the nearest point? If a colony is huge, it might have a center very far from you, but an engulfing edge very close.

Reply:
Optimally the AI would know about colonies. I just fixed this very serious problem: if using Translation the AI thinks all colony members are near the origin, because Translation is relative to the colony parent. So I just kinda plugged the massive problem here.
Opened a separate issue about making the AI know about colonies: https://github.com/Revolutionary-Games/Thrive/issues/2348

Reply:
ohh, yeah, that would definitely break things. Makes sense to me then.